File: Iteration2/SourceCode/src/main/java/CourseSectionRegister.java
public class CourseSectionRegister {
        CourseSection courseSection;
        String status;
        Student student;
        CourseSectionRegister(CourseSection courseSection,String status,Student student){
            this.courseSection=courseSection;
            this.status=status;
            this.student=student;
            Schedule StudentSchedule=this.student.getSchedule();
            StudentSchedule.addToSchedule(this);
        }


    public CourseSection getCourseSection() {
        return courseSection;
    }

    public void setCourseSection(CourseSection courseSection) {
        this.courseSection = courseSection;
    }

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }
}

File: Iteration3/SourceCode/CourseSectionRegister.py
class CourseSectionRegister(object):
    def __init__(self, courseSection, status, student):

        self.__courseSection=courseSection
        self.__status=status
        self.__student=student
        StudentSchedule =self.__student.getSchedule()
        StudentSchedule.addToSchedule(self)


    def getCourseSection(self):
        return self.__courseSection

    def setCourseSection(self, courseSection):
        self.__courseSection = courseSection

    def getStatus(self):
        return self.__status

    def setStatus(self, status):
        self.__status = status
